packages/admin/admin/src/form/FinalFormChipSelect.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { FieldRenderProps } from "react-final-form";
+
+import { ChipSelect, ChipSelectProps } from "./ChipSelect";
+
+export type FinalFormChipSelectProps<T> = Omit<ChipSelectProps<T>, "onChange"> & FieldRenderProps<T, HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
`HTMLTextAreaElement` is probably incorrect. 
And I think it should be `HTMLSelectElement` instead of `HTMLInputElement`. 

```suggestion
export type FinalFormChipSelectProps<T> = Omit<ChipSelectProps<T>, "onChange"> & FieldRenderProps<T, HTMLSelectElement>;
```

Reply:
If i change it to `HTMLSelectElement`:
<img width="1417" alt="image" src="https://github.com/vivid-planet/comet/assets/56400587/a0670fd3-e17a-4285-9311-95cbd36b2129">